lib/rom/http/dataset.rb

```diff
@@ -11,15 +13,15 @@ class Dataset
 
       include Enumerable
       include Dry::Equalizer(:config, :options)
-      include ROM::Options
+      extend Initializer
```

Comment:
Can we make this explicitly `extend ::ROM::Initializer`, `Initializer` seems like it could be a fairly common global constant name